### ISSUE TITLE
github clone issue during inventaire-client installation

### ISSUE THREAD

**Issue by niamtokik** [inventaire/inventaire-client#59]
Hi everyone,

Just a small bug:

Here the issue (after `npm run install-client`):

```
Clonage dans './public/i18n'...
The authenticity of host 'github.com (192.30.252.128)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
+---[RSA 2048]----+
| =+o...+=o..     |
|o++... *o .      |
|*.o.  *o.        |
|oo.  ..o.= .     |
|.+o. .. S =      |
|*=+ .  o = .     |
|OE .  . o        |
| o     .         |
|                 |
+----[SHA256]-----+
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added 'github.com,192.30.252.128' (RSA) to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.
```

Here the patch:

``` diff
diff --git a/scripts/postinstall b/scripts/postinstall
index c0651b0..eb496da 100755
--- a/scripts/postinstall
+++ b/scripts/postinstall
@@ -12,7 +12,7 @@ npm run patches
 # need to build once to make the public folder available
 ./node_modules/brunch/bin/brunch build

-git clone -b i18n git@github.com:maxlath/inventaire-client.git ./public/i18n
+git clone -b i18n https://github.com/maxlath/inventaire-client.git ./public/i18n
 mkdir -p ./server/lib/emails/i18n/dist
 ./scripts/update_i18n_all
```

I don't know if github over ssh is desired feature, but, its cause error during installation. I think using https instead of ssh its better.

Please tell me if I can do a PR on that. :)

**Comment by maxlath** [comment]
indeed, this has already been done on the server repo (see https://github.com/inventaire/inventaire/issues/36 and https://github.com/inventaire/inventaire/commit/0c7d575dadb2f687fbd35095d8e569c0866ebabe ), but we forgot the client repo: PR welcome!

**Comment by maxlath** [comment]
fixed by 6e9a7aee
